Library/Formula/viewvc.rb

```diff
@@ -0,0 +1,36 @@
+class Viewvc < Formula
+  desc "Browser interface for CVS and Subversion repositories"
+  homepage "http://www.viewvc.org"
+  url "http://viewvc.tigris.org/files/documents/3330/49392/viewvc-1.1.23.tar.gz"
+  sha256 "d14290b23f44e8f5301a7ca9fc946bcd0f8bb8206d9b9c111ac56e19f2588b7d"
+
+  if Formula["python"].installed?
```

Comment:
`viewvc-install` replaces the shebang lines in its bin scripts with whatever python it was invoked with. So, their installation of viewvc would break.